### DISCUSSION TITLE
Ruler w/o mouse

### DISCUSSION THREAD

**Discussion by stradichenko** [ahrm/sioyek#811] [Q&A] [answered]
Is very silly; how do you guys Enter Ruler at an specific line without using the right click? 
My workflow would be to Start reading with ruler; make a highlight in Visual mode and then go back to ruler.

Have a great day!

**Comment by ahrm** [comment, marked as answer]
If you want to return to the last line in ruler mode, you can press `` (double tap the tick button).

**Reply by stradichenko** [reply]
Thank you; I'm just getting the hang of sioyek but loving every minute.
